src/has160_fmt_plug.c

```diff
@@ -73,10 +73,13 @@ static void init(struct fmt_main *self)
 #ifdef _OPENMP
 	int omp_t;
 
-	omp_t = omp_get_max_threads();
-	self->params.min_keys_per_crypt *= omp_t;
-	omp_t *= OMP_SCALE;
-	self->params.max_keys_per_crypt *= omp_t;
+	int omp_t = omp_get_max_threads();
```

Comment:
Looks like duplicate definition of `omp_t` - does this even build?

Reply:
Fixed now. Yes, the build did succeed earlier.